dom0/sd-clean-all.sls

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+set-fedora-as-default-dispvm:
+  cmd.run:
+    - name: qubes-prefs default_dispvm fedora-30-dvm
+
+remove-dom0-sdw-config-files:
+  file.absent:
+    - names:
+      - /opt/securedrop
+      - /etc/yum.repos.d/securedrop-workstation-dom0.repo
+      - /usr/bin/securedrop-update
+      - /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation-test
+      - /etc/cron.daily/securedrop-update-cron
+      - /usr/share/securedrop/icons
+
+sd-cleanup-sys-firewall:
+  cmd.run:
+    - names:
+      - qvm-run sys-firewall 'sudo rm -f /rw/config/RPM-GPG-KEY-securedrop-workstation-test'
+      - qvm-run sys-firewall 'sudo rm -f /rw/config/sd-copy-rpm-repo-pubkey.sh'
+      - qvm-run sys-firewall 'sudo rm -f /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation-test'
+      - qvm-run sys-firewall 'sudo perl -pi -E "s#^/rw/config/sd-copy-rpm-repo-pubkey.sh##" /rw/config/rc.local'
+
+sd-cleanup-rpc-mgmt-policy:
+  file.replace:
+    - names:
+      - /etc/qubes-rpc/policy/qubes.VMShell
```

Comment:
these policies do not exist on my machine. How/when are these created? qubesctl?

Reply:
These policies are created automatically during upgrades via a disp-mgmt VM, and also removed automatically. If someone ctrl+c's the `make all` process, it's possible for them to hang around. Possibly overkill to worry about them here, I'm happy to excise.